src/zmqpp/message.cpp

```diff
@@ -211,6 +211,12 @@ void message::get(std::string& string, size_t const part) const
 	string.assign( get(part) );
 }
 
+void message::get(size_t& unsigned_long, size_t const part) const
+{
+	uint64_t u64 = static_cast<uint64_t>(unsigned_long);
+	get(u64, sizeof(u64));
```

Comment:
This needs to be inverted I think. Look like it would get the value of the unsigned_long to the the u64 then get the value from the message.